### ISSUE TITLE
jackaudio not working on m-audio's connectiv and fasttrack pro 

### ISSUE THREAD

**Issue by cjdg** [jackaudio/jack2#205]
jack is not working with a couple of interfaces that i own both exhibits the same and after that i cannot use jack until i reboot the computer i get the following errors:

16:32:30.159 Registro iniciado --- vie. may 6 16:32:30 2016 ---
16:32:31.790 D-BUS: El servidor JACK no puede iniciarse. Disculpa
Cannot connect to server socket err = No existe el archivo o el directorio
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for 4294967295, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for 4294967295, skipping unlock
Fri May  6 16:32:31 2016: Starting jack server...
Fri May  6 16:32:31 2016: Jack: Server `default' registered
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::InitConnections size = 6522944 
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::InitClients
Fri May  6 16:32:31 2016: JACK server starting in realtime mode with priority 10
Fri May  6 16:32:31 2016: self-connect-mode is "Don't restrict self connect requests"
Fri May  6 16:32:31 2016: Jack: JackShmMem::new index = 0 attached = 8ab92000 size = 82274202 
Fri May  6 16:32:31 2016: Jack: JackShmMem::new placement size = 13047706
Fri May  6 16:32:31 2016: Jack: Succeeded in locking 82274202 byte memory area
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::InitConnections size = 6522944 
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::InitClients
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::InitConnections size = 6522944 
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::InitClients
Fri May  6 16:32:31 2016: Jack: JackShmMem::new index = 1 attached = 99029000 size = 1186 
Fri May  6 16:32:31 2016: Jack: Succeeded in locking 1186 byte memory area
Fri May  6 16:32:31 2016: Jack: JackPosixThread::StartImp : create non RT thread
Fri May  6 16:32:31 2016: Jack: JackPosixThread::ThreadHandler : start
Fri May  6 16:32:31 2016: Jack: playback device hw:0
Fri May  6 16:32:31 2016: Jack: capture device hw:0
Fri May  6 16:32:31 2016: Jack: capture device hw:0
Fri May  6 16:32:31 2016: Jack: playback device hw:0
Fri May  6 16:32:31 2016: Jack: apparent rate = 44100
Fri May  6 16:32:31 2016: Jack: frames per period = 1024
Fri May  6 16:32:31 2016: Jack: JackDriver::Open capture_driver_name = hw:0
Fri May  6 16:32:31 2016: Jack: JackDriver::Open playback_driver_name = hw:0
Fri May  6 16:32:31 2016: Jack: Check protocol client = 8 server = 8
Fri May  6 16:32:31 2016: Jack: JackEngine::ClientInternalOpen: name = system
Fri May  6 16:32:31 2016: Jack: JackEngine::AllocateRefNum ref = 0
Fri May  6 16:32:31 2016: Jack: JackPosixSemaphore::Allocate name = jack_sem.1012_default_system val = 0
Fri May  6 16:32:31 2016: Jack: JackEngine::NotifyAddClient: name = system
Fri May  6 16:32:31 2016: Jack: JackGraphManager::SetBufferSize size = 1024
Fri May  6 16:32:31 2016: Jack: JackConnectionManager::DirectConnect first: ref1 = 0 ref2 = 0
Fri May  6 16:32:31 2016: Jack: JackGraphManager::ConnectRefNum cur_index = 0 ref1 = 0 ref2 = 0
Fri May  6 16:32:31 2016: Jack: JackDriver::SetupDriverSync driver sem in flush mode
Fri May  6 16:32:31 2016: ERROR: cannot register object path "/org/freedesktop/ReserveDevice1/Audio0": A handler is already registered for /org/freedesktop/ReserveDevice1/Audio0
Fri May  6 16:32:31 2016: ERROR: Failed to acquire device name : Audio0 error : A handler is already registered for /org/freedesktop/ReserveDevice1/Audio0
Fri May  6 16:32:31 2016: ERROR: Audio device hw:0 cannot be acquired...
Fri May  6 16:32:31 2016: Jack: ~JackDriver
Fri May  6 16:32:31 2016: ERROR: Cannot initialize driver
Fri May  6 16:32:31 2016: Jack: no message buffer overruns
Fri May  6 16:32:31 2016: Jack: JackPosixThread::Stop
Fri May  6 16:32:31 2016: Jack: JackPosixThread::ThreadHandler : exit
Fri May  6 16:32:31 2016: ERROR: JackServer::Open failed with -1
Fri May  6 16:32:31 2016: Jack: Succeeded in unlocking 82274202 byte memory area
Fri May  6 16:32:31 2016: Jack: JackShmMem::delete size = 0 index = 0
Fri May  6 16:32:31 2016: Jack: ~JackDriver
Fri May  6 16:32:31 2016: Jack: Succeeded in unlocking 1186 byte memory area
Fri May  6 16:32:31 2016: Jack: JackShmMem::delete size = 0 index = 1
Fri May  6 16:32:31 2016: Jack: Cleaning up shared memory
Fri May  6 16:32:31 2016: Jack: Cleaning up files
Fri May  6 16:32:31 2016: Jack: Unregistering server `default'
Fri May  6 16:32:31 2016: ERROR: Failed to open server
16:32:33.932 No puede conectarse al servidor JACK como cliente. - La operación global falló. - No puede conectarse al servidor. Por favor revise la ventana de mensajes para mas información.
Cannot connect to server socket err = No existe el archivo o el directorio
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for 4294967295, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for 4294967295, skipping unlock
Fri May  6 16:32:33 2016: Saving settings to "/home/cjdg/.config/jack/conf.xml" ...

**Comment by mseeber** [comment]
[...]
`May 6 16:32:31 2016: ERROR: cannot register object path "/org/freedesktop/ReserveDevice1/Audio0": A handler is already registered for /org/freedesktop/ReserveDevice1/Audio0 Fri May 6 16:32:31 2016: ERROR: Failed to acquire device name : Audio0 error : A handler is already registered for /org/freedesktop/ReserveDevice1/Audio0 Fri May 6 16:32:31 2016: ERROR: Audio device hw:0 cannot be acquired... Fri May 6 16:32:31 2016: Jack: ~JackDriver Fri May 6 16:32:31 2016: ERROR: Cannot initialize driver Fri May 6 16:32:31 2016: Jack: no message buffer overruns Fri May 6 16:32:31 2016: Jack: JackPosixThread::Stop`
[...]

Another application seems to have reserved the device.

**Comment by cjdg** [comment]
not really, first i restart the computer and there is no issue, it works when i put the default options but when i want to put 4 in 4 outs is when it crash
